<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoleculeWindow</class>
 <widget class="QMainWindow" name="MoleculeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayoutMain"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileNewAction"/>
    <addaction name="fileOpenAction"/>
    <addaction name="fileSaveAction"/>
    <addaction name="fileSaveAsAction"/>
    <addaction name="separator"/>
    <addaction name="fileEditMolFileAction"/>
    <addaction name="separator"/>
    <addaction name="fileExitAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="helpHelpAction"/>
    <addaction name="helpAboutAction"/>
    <addaction name="helpAboutQtAction"/>
   </widget>
   <widget class="QMenu" name="menuStructure">
    <property name="title">
     <string>Structure</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="moleculeExportImageAction"/>
    </widget>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="viewMoleculeAction"/>
    <addaction name="viewSummaryAction"/>
    <addaction name="separator"/>
    <addaction name="viewViewerSettingsAction"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="windowsCascadeAction"/>
    <addaction name="windowsTileAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menuStructure"/>
   <addaction name="menuView"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileNewAction"/>
   <addaction name="fileOpenAction"/>
   <addaction name="fileSaveAction"/>
   <addaction name="fileSaveAsAction"/>
   <addaction name="fileEditMolFileAction"/>
   <addaction name="separator"/>
   <addaction name="viewMoleculeAction"/>
   <addaction name="viewSummaryAction"/>
   <addaction name="separator"/>
   <addaction name="viewViewerSettingsAction"/>
   <addaction name="separator"/>
   <addaction name="moleculeExportImageAction"/>
   <addaction name="separator"/>
   <addaction name="helpHelpAction"/>
  </widget>
  <action name="fileOpenAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/fileOpenAction.png</normaloff>:/icons/16x16/fileOpenAction.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileNewAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/fileNewAction.png</normaloff>:/icons/16x16/fileNewAction.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="helpHelpAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/helpHelpAction.png</normaloff>:/icons/16x16/helpHelpAction.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="moleculeExportImageAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/moleculeExportImageAction.png</normaloff>:/icons/16x16/moleculeExportImageAction.png</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="fileExitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="helpAboutQtAction">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="helpAboutAction">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="fileEditMolFileAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/fileEditMolFileAction.png</normaloff>:/icons/16x16/fileEditMolFileAction.png</iconset>
   </property>
   <property name="text">
    <string>Edit *.mol file...</string>
   </property>
  </action>
  <action name="fileSaveAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/fileSaveAction.png</normaloff>:/icons/16x16/fileSaveAction.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fileSaveAsAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/fileSaveAsAction.png</normaloff>:/icons/16x16/fileSaveAsAction.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="viewMoleculeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/viewMoleculeAction.png</normaloff>:/icons/16x16/viewMoleculeAction.png</iconset>
   </property>
   <property name="text">
    <string>Molecule</string>
   </property>
  </action>
  <action name="viewSummaryAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/viewSummaryAction.png</normaloff>:/icons/16x16/viewSummaryAction.png</iconset>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
  </action>
  <action name="windowsCascadeAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/windowsCascadeAction.png</normaloff>:/icons/16x16/windowsCascadeAction.png</iconset>
   </property>
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="windowsTileAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/windowsTileAction.png</normaloff>:/icons/16x16/windowsTileAction.png</iconset>
   </property>
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="viewViewerSettingsAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/16x16/viewViewerSettingsAction.png</normaloff>:/icons/16x16/viewViewerSettingsAction.png</iconset>
   </property>
   <property name="text">
    <string>Viewer Settings...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
